pages/getting-started/first-steps-with-docker.mdx

```diff
@@ -0,0 +1,522 @@
+
+```
+docker system info
+```
+
+If Docker is running correctly, this command will return system-wide information. If not, you may receive an error indicating that Docker is not running.
+
+</Steps>
+
+### Troubleshooting
+
+If you encounter any errors or if Docker does not seem to be running:
+
+- Ensure that the Docker Desktop application (for Mac and Windows) is running.
+- On Linux, you might need to start the Docker service manually using a command like `sudo service docker start` or `sudo systemctl start docker`, depending on your Linux distribution.
+
+## Managing Docker Service on Windows using PowerShell
+
+On Windows, Docker typically runs as a service via Docker Desktop. While Docker
+Desktop provides a user-friendly interface for managing Docker, there are
+instances where you might want to control the Docker service directly from the
+command line using PowerShell.
+
+Here's how you can manage the Docker service on Windows using PowerShell:
+
+### Open PowerShell as an administrator
+
+- Right-click the Start button and select "Terminal (Admin)" or search
+  for "PowerShell" in the start menu, right-click "Windows PowerShell", and
+  select "Run as administrator".
+
+### Start the Docker Service
+
+If Docker isn't running and you want to start it, use the following command:
+
+```powershell
+net start com.docker.service
+```
+
+You should receive a message indicating that the Docker service has started successfully.
+
+### Stop the Docker Service
+
+To stop the Docker service, use the following command:
+
+```powershell
+net stop com.docker.service
+```
+
+You should receive a message indicating that the Docker service has stopped.
+
+### Check the Status of the Docker Service
+
+To verify the status of the Docker service, whether it's running or stopped, use:
+
+```powershell
+Get-Service com.docker.service
+```
+
+This command will display the status of the Docker service.
+
+## Work with Docker
+
+If you are new to Docker, this guide will help you get a grasp of Docker and
+make it easier to accomplish tasks within Memgraph. After installing Docker, all
+commands are run from the command-line tool of your preference.
+
+### Download the Memgraph Docker image
+
+You can think about downloading Memgraph Docker images as obtaining a blueprint
+for Memgraph, complete with all its core features. This image is your starting
+point. From it, you can launch a container, which acts as a live instance of
+Memgraph. All your interactions, queries, and transactions with Memgraph occur
+within this container. This ensures a consistent and isolated setting for all
+your database tasks. From a single image, you can spawn multiple containers.
+Each container functions as a distinct Memgraph instance.
+
+Images are downloaded using the `docker pull` command followed by the name of
+the Docker image. We encourage you to use Memgraph Platform as it includes
+everything you might need while making the best of Memgraph.
+
```

Comment:
I would expand with (just use the normal links, this is how it C/P from the website)

Besides the database, it also includes all the tools you might need to analyze your data, such as command-line interface [mgconsole](https://memgraph.com/docs/getting-started/cli), web interface [Memgraph Lab](https://memgraph.com/docs/data-visualization) and a complete set of algorithms within a [MAGE](https://memgraph.com/docs/advanced-algorithms) library

Reply:
Done.